langserver/handlers/execute_command.go

```diff
@@ -7,26 +7,45 @@ import (
 	"strings"
 
 	"github.com/creachadair/jrpc2/code"
+	lsctx "github.com/hashicorp/terraform-ls/internal/context"
 	lsp "github.com/sourcegraph/go-lsp"
 )
 
 type executeCommandHandler func(context.Context, commandArgs) (interface{}, error)
 type executeCommandHandlers map[string]executeCommandHandler
 
+const langServerPrefix = "terraform-ls."
+
 var handlers = executeCommandHandlers{
-	"rootmodules": executeCommandRootModulesHandler,
+	prefix("rootmodules"): executeCommandRootModulesHandler,
+}
+
+func prefix(name string) string {
```

Comment:
Nitpick: can we call this something like `prefixedCommandName`?

Reply:
I shall but I'm keeping it present tense `prefixCommandName()`